CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Ephemeral Namespace Operator Local Development
+
+### Purpose
+The purpose of this doc is to deploy the Ephemeral Namespace Operator (ENO) in Minikube for local development.  
+
+### Requirements
+- Clone the [Clowder](https://github.com/RedHatInsights/clowder) repository
+- Install [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- Login credentials for [Quay](https://quay.io)
+- Install [Bonfire](https://pypi.org/project/crc-bonfire/)
+
+### Setup Minikube
+```
+minikube start --cpus 4 --disk-size 36GB --memory 8192MB --addons registry --addons ingress --addons metrics-server
+```
+
+### Retrieve Quay Pull Secret
+The following steps with walk you through getting your quay pull secret:
+- Login to [Quay](https://quay.io)
+- Click on your username at the top right of the homepage and click `Account Settings`
+- At the `CLI Password` section, click `Generate Encrypted Password`
+- Click `Kubernetes Secret`
+- Click `Download <your username>-secret.yml`
+
+### Setup Ephemeral Base Namespace
+The `ephemeral-base` namespace is used for copying secrets to the ephemeral namespaces that are created using the ENO.
+
+Run the following command to create the namespace:
+```
+oc create ns ephemeral-base --dry-run=client -o yaml | oc apply -f -
+```
+
+Apply the pull secret yaml file you retrieved from Quay:
+```
+oc apply -f ~/path/to/<your username>-secret.yml -n ephemeral-base
+```
+
+### Install Clowder
+Change to the Clowder directory and run:
+```
+./build/kube_setup.sh && make deploy-minikube-quick
+```
+
+### Run the Ephemeral Namespace Operator
+Change to the ENO directory and apply the following CRD:
+```
+oc apply -f config/crd/static/cloud.redhat.com_frontendenvironments.yaml
+```
+
+Start the ENO:
```

Comment:
It may be useful here to add a little bit of verbiage around what this command actually does. Explaining that it doesn't actually install or run the operator on the cluster and will not est things like the container sizing, but will literally only do a functional code test.